Scotch.Tests/Scotch.Tests.csproj

```diff
@@ -1,14 +1,28 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Is this backwards compatible with net5.0? Should we be targeting both?

Reply:
Resolved in [fb7f153](https://github.com/EasyPost/scotch/pull/1/commits/fb7f15380aa31ddb1f7b2ab77e2b4677e1d5a0d1)